fastlane/lib/fastlane/lane_manager_base.rb

```diff
@@ -75,5 +75,14 @@ def self.print_lane_context
         rows: FastlaneCore::PrintTable.transform_output(rows)
       }))
     end
+
+    def self.print_error_line(ex)
+      error_line = ex.backtrace[0].match("Fastfile:(\\d+):")
```

Comment:
`backtrace[0]` is not always set for some reason and fastlane throws exception:

fastlane-2.96.1/fastlane/lib/fastlane/lane_manager_base.rb:80:in `print_error_line': [!] undefined method `match' for nil:NilClass (NoMethodError)

Null check?